libs/shared/ui/src/lib/components/tag/tag.tsx

```diff
@@ -13,7 +13,7 @@ export interface TagProps {
 export function Tag(props: TagProps) {
   const { children, className = '', size = TagSize.NORMAL, dataTestId = 'tag' } = props
 
-  const normalClassName = size === TagSize.NORMAL ? 'h-6 px-2' : ''
+  const normalClassName = size === TagSize.NORMAL ? 'h-7 px-2' : ''
```

Comment:
I check the figma and everywhere I was finding 28px for tags so I updated it here directly